### DISCUSSION TITLE
请求返回500错误，日志显示ReadRespHeaderError

### DISCUSSION THREAD

**Discussion by jyuan68** [bfenetworks/bfe#1131] [Q&A (问题和回答)]
服务突然报500，access.log看是500错误，错误是："error":"BK_READ_RESP_HEADER,ReadRespHeaderError: EOF"。

请问，这个错误代表什么意思，可能是什么原因？？

**Comment by mileszhang2016** [comment]
这个错误的意思是：从后端的RS读取Reponse Header错误。
可以看看后端的服务是否正常
